implementations/elixir/ockam/ockam/lib/ockam/transport/tcp/listener.ex

```diff
@@ -32,21 +32,15 @@ if Code.ensure_loaded?(:ranch) do
       end
     end
 
+    # https://github.com/ninenines/ranch/blob/2.1.0/src/ranch_listener_sup.erl#L21-L22
```

Comment:
I think this comment is more useful as a PR comment than code comment. It helped me understand why we got rid of the case block, but not sure how useful it will be for someone reading code going forward

Reply:
That's very fair. I'd also included it in the commit message already, so I've dropped the code comment as part of a49c2e8b1e1fe158894b14b1856c039d3ec4040a.